README.md

```diff
@@ -13,3 +13,11 @@ FLexible And REliable Fitting LibrarY is a library to perform fits to distributi
 # Documentation
 
 [https://flarefly.github.io/flarefly/index.html](https://flarefly.github.io/flarefly/index.html)
+
+## Tutorial
+
+If you want to get familiar with flarefly, the following tutorial is available:
+
+| Type | Link |
+| -------------- | ------------- |
+| Invariant mass fit |  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/AlexBigO/flarefly/blob/myBranch/tutorials/flarefly_tutorial_Bzero.ipynb) |
```

Comment:
I would specify that it is a tutorial for a binned fit (we will include also the one for the unbinned fit sooner or later ;)) and I would already put it in `main`, with the idea of having soon a new release
```suggestion
| Binned invariant mass fit |  [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/flarefly/flarefly/blob/main/tutorials/flarefly_tutorial_Bzero.ipynb) |
```